src/components/LoanFinding/LendingCategorySection.vue

```diff
@@ -170,6 +170,7 @@ export default {
 
 				this.$refs[key][0].addToBasket(amount);
 			}
+			this.$kvTrackEvent('loan-card', 'add-to-basket', 'relending-lending-home-add-all');
```

Comment:
Actually it would probably be good to add two more props, the comma-separated list of loan IDs, and the comma-separated list of loan amounts.

Reply:
Nice suggestion!